locale/de/prefwindow.dtd

```diff
@@ -4,12 +4,12 @@
 <!ENTITY advanced.label "Erweitert">
 <!ENTITY about.label "Über">
 <!ENTITY copyright.label "Copyright © 2012,2013,2015 Jonathan Kamens">
-<!ENTITY license.label "Released under the Mozilla Public License, version 2.0">
-<!ENTITY support.label "Bitte unterstützen Sie die Entwicklung dieses und anderen tollen Add-Ons!">
+<!ENTITY license.label "Veröffentlicht unter der Mozilla Public License, version 1.1">
```

Comment:
Done, see 2c9b7a7 below. You might want to change the number on the BabelZilla WTS page as well.